config/install/user.role.authenticated.yml

```diff
@@ -7,13 +7,15 @@ dependencies:
     - eck_bundle_permissions
     - form_mode_control
     - system
+    - workflow
 id: authenticated
 label: 'Authenticated user'
 weight: 1
 is_admin: false
 permissions:
   - 'access content'
   - 'access user profiles'
+  - 'create school_registration_verification workflow_transition'
```

Comment:
This is still needed? As we now have a new role.

Reply:
Yes, this is added by the module, so that this role can participate in workflow.